.github/workflows/continous-integration-docker.yml

```diff
@@ -66,23 +66,72 @@ env:
   GAR_BASE: us-docker.pkg.dev/zealous-zebra/zebra
 
 jobs:
-  changed-files:
+  get-available-disks:
```

Comment:
Update the .patch. one too?

Reply:
This change needs to be made in PR #4385, @gustavovalverde can you do that?